src/main/java/net/sacredlabyrinth/phaed/simpleclans/managers/SettingsManager.java

```diff
@@ -443,6 +443,7 @@ public enum ConfigField {
         CLAN_MAX_DESCRIPTION_LENGTH("clan.max-description-length", 120),
         CLAN_MIN_DESCRIPTION_LENGTH("clan.min-description-length", 10),
         CLAN_MAX_MEMBERS("clan.max-members", 25),
+        CLAN_UNVERIFIED_MAX_MEMBERS("clan.unverified-max-members", 25),
```

Comment:
So is that [10](https://github.com/RoinujNosde/SimpleClans/pull/422/files#diff-58cdd3d308ccba6c594e040ff9c065bb11eeb6e30f35ba87694ea45d5ae6096cR151) or 25?

Reply:
oops